src/test/log_console_test.cpp

```diff
@@ -48,6 +48,8 @@ TEST(LogStaticTest, LogInitialized)
 {
     std::stringstream out;
 
+    // First shut down the logging in case another test has already initialized logging.
+    Log::Shutdown();
```

Comment:
Isn't it a problem for the LogShutdown test case below as well? Perhaps it's better to create a LogStaticTest test fixture that would just call Log::Shutdown() in the SetUp?

Reply:
Right, I did not think of what would happen if the log was already initialized with a different stream. I'll try to fix it up tomorrow.